index.html

```diff
@@ -257,10 +257,10 @@ <h3>Ecosystem Overview</h3>
 <a>verifiable presentation</a> that proves the holder possesses the required verifiable credentials with certain characteristics.
 Examples of verifiers include employers, security personnel, and websites.
         </dd>
-        <dt><a>identifier registry</a></dt>
+        <dt><a>Verifiable data registry</a></dt>
```

Comment:
Concrete suggestion: Change to "Cryptographic Data Registry" 

Reasoning: The data isn't always verifiable (e.g. federated or DHT-style systems). In almost every case, the registry does contain data that is used in cryptographic processes of one kind or another.

Reply:
unless there is significant consensus for "Cryptographic Data Registry," or a lot of opposition to "Verifiable data registry" I would prefer to keep "Verifiable data registry" for now.

Reply:
This would mean that a "DID Ledger" is a type of "Verifiable Data Registry"... which conflates the way "Verifiable" is used in the VC data model spec, and the general usage of the word. It doesn't feel quite right to me because some DID Ledgers, like Veres One, store data (like service descriptions, that aren't "Verifiable" in the "Verifiable Credentials" sense of the word, so it's a strange fit. 

I think the bigger issue is that a federated/mirrored corporate system would also be a "Verifiable Data Registry", but some of that information is not verifiable (i.e., it is only valid because the corporation says it's valid)... but the fact that all of these systems hold cryptographic data in one form or another is the common denominator here.

Keep in mind that this started out as an "Identifier Registry" because the block had a specific purpose. We are now expanding that purpose and have to be careful to not expand too far, or pick the wrong word.

We need more input on this, perhaps on a call (because I don't think the people that may have something to say about this are paying attention to code commit comments), perhaps via a run off poll because we're bike shedding at this point and it'll go on forever w/o some finality to the name change. We had to bike shed this name early on and did an IRV poll to get finality that time. We should do this again so we can get this PR sorted quickly and into the spec.

Reply:
Verifiable Data Registry is inaccurate. The credential itself contains the verifiable data.

I think this registry potentially contains a broad variety of data that is used for verification, such as service descriptions, schemas, credential definitions, DIDs, revocation information, etc. 

How does the name Verification Material Registry sound? I think it reflects the scope and intent for which it will be used.

Reply:
"Verification Material Registry" is probably the most accurate thing so far, so I'd be fine w/ that ... and now we're squarely in "we need to ask the WG" territory. I'll try to draft up an Instant Runoff Vote for this... We have "Verifiable Data Registry", "Cryptographic Data Registry", "Cryptographic Material Registry", and "Verification Material Registry"... any other contenders that folks want to put forward?

Reply:
Verification material registry is fine with me as well.

Reply:
I'm a little concerned that we're perhaps losing the original main intent for the registry which was to be an identifier registry. I understand that there are other registry needs as well (e.g. Sovrin credential schemas) -- and that many of them are related to verification. Perhaps we can just call this system component "Registries" and then talk about what different types of registries might be out there?

What's important, IMO, is that they are seen as independent components in the ecosystem, i.e. they are not tightly coupled to the issuer, verifier, or holder, allowing for agility and greater decentralization.